apps/gpsinfo/gps-info.js

```diff
@@ -109,8 +109,13 @@ function onGPS(fix) {
 }
 
 function onGPSraw(nmea) {
+  var nofBD = 0;
+  var nofGP = 0;
```

Comment:
nofBD and nofGP should be global, otherwise one of them is always zero.

Reply:
Thanks, resolved with commit aa45ecd .